documentation/ipi-install/modules/ipi-install-troubleshooting-cleaning-up-previous-installations.adoc

```diff
@@ -25,6 +25,8 @@ do
   sudo virsh undefine $i;
   sudo virsh vol-delete $i --pool default;
   sudo virsh vol-delete $i.ign --pool default;
+  sudo virsh pool-destroy default;
+  sudo virsh pool-undefine default;
 done
 ----
 
```

Comment:
[root@worker-0 ~]# sudo virsh pool-list 
setlocale: No such file or directory
 Name                   State    Autostart
--------------------------------------------
 default                active   yes
 kni4-25rhx-bootstrap   active   yes
 kni4-27qxs-bootstrap   active   yes
 kni4-2rrwq-bootstrap   active   yes
 kni4-2tqbn-bootstrap   active   yes
 kni4-4rsbl-bootstrap   active   yes
 kni4-54hr9-bootstrap   active   yes
 kni4-6mz49-bootstrap   active   yes
 kni4-76dds-bootstrap   active   yes
 kni4-76gvw-bootstrap   active   yes
 kni4-7bbq9-bootstrap   active   yes
 kni4-7xfvk-bootstrap   active   yes
 kni4-8fg2k-bootstrap   active   yes
 kni4-b6l7v-bootstrap   active   yes
 kni4-b9m89-bootstrap   active   yes
 kni4-c4jf6-bootstrap   active   yes
 kni4-d4pb8-bootstrap   active   yes
 kni4-grl68-bootstrap   active   yes
 kni4-hscxh-bootstrap   active   yes
 kni4-jfvhd-bootstrap   active   yes
 kni4-jhthd-bootstrap   active   yes
 kni4-jnv6z-bootstrap   active   yes
 kni4-jrmgk-bootstrap   active   yes
 kni4-kgzvx-bootstrap   active   yes
 kni4-ks4b7-bootstrap   active   yes
 kni4-l2vr7-bootstrap   active   yes
 kni4-ljrkk-bootstrap   active   yes
 kni4-nfzv6-bootstrap   active   yes
 kni4-nq9f8-bootstrap   active   yes
 kni4-q6r9t-bootstrap   active   yes
 kni4-rl8zr-bootstrap   active   yes
 kni4-rqw52-bootstrap   active   yes
 kni4-td6f6-bootstrap   active   yes
 kni4-v4n5r-bootstrap   active   yes
 kni4-v6b7t-bootstrap   active   yes
 kni4-vdmn2-bootstrap   active   yes
 kni4-vfx58-bootstrap   active   yes
 kni4-vz976-bootstrap   active   yes
 kni4-wjwvs-bootstrap   active   yes
 kni4-wr9tg-bootstrap   active   yes
 kni4-wtbwv-bootstrap   active   yes
 kni4-xx5fs-bootstrap   active   yes
 kni4-zcn58-bootstrap   active   yes

Reply:
We need a conditional then here...